media-libs/gavl/gavl-1.4.0-r2.ebuild

```
@@ -16,7 +16,11 @@ IUSE="doc"
 
 BDEPEND="doc? ( app-doc/doxygen )"
 
-PATCHES=( "${FILESDIR}/${PV}-x32.diff" )
+PATCHES=(
+	"${FILESDIR}/${PV}-x32.patch"
+	"${FILESDIR}/${PN}-${PV}-gnu_source.patch"
```

```suggestion
	"${FILESDIR}/${P}-gnu_source.patch"
```

But match existing style instead:
```suggestion
	"${FILESDIR}/${PV}-gnu_source.patch"
```

and rename the patch.